package/MDAnalysis/analysis/contacts.py

```diff
@@ -216,7 +219,7 @@ def is_any_closer(r, r0, dist=2.5):
 from MDAnalysis.lib.util import openany
 from MDAnalysis.analysis.distances import distance_array
 from MDAnalysis.core.groups import AtomGroup
-from .base import AnalysisBase
+from .base import AnalysisBase, Results
```

Comment:
Remove the import here

Reply:
oops 🙈 I thought I had done that after your other comment on DensityAnalysis